spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/flyway/FlywayProperties.java

```diff
@@ -73,6 +73,12 @@
 	 */
 	private Integer lockRetryCount;
 
+	/**
+	 * Whether or not Flyway should try to automatically detect SQL migration file
```

Comment:
We use "Whether Flyway" everywhere else in this class. Can you please adapt for consistency?